mods/ra/maps/allies-01/map.yaml

```diff
@@ -539,9 +539,12 @@ Rules:
 			GenericVisibility: Enemy, Ally, Neutral
 			GenericStancePrefix: false
 			ShowOwnerRow: false
-	^CivInfantry:
+	^ArmedCivilian:
 		RevealsShroud:
 			Range: 0c0
+		AutoTarget:
```

Comment:
Because the second civilian is a `C7` (which already is an `^ArmedCivilian`), so I only need to have that line once ;)